docs/validation.md

```diff
@@ -46,10 +46,20 @@ $myValidation->setName('John Doe');
 Validation::persist($myValidation);
 
 // Activate the validation object
+// As the email content isn't defined, nothing will be sent
 $myValidation->send();
 
 // Retrieve the validation link
 $myLink = $myValidation->getLink();
+
+// Define the content of the email if you would like to send emails through Penneo as well
```

Comment:
Could you elaborate on this to make it very clear that this applies if you want to __re-send__ the signing request __through__ Penneo at a later point.
I'm a pain in the ass, I know ;-)